cli/src/commands/schema/dump.ts

```diff
@@ -0,0 +1,35 @@
+import { Flags } from '@oclif/core';
+import { getCurrentBranchDetails } from '@xata.io/client';
+import { BaseCommand } from '../../base.js';
+import fetch from 'node-fetch';
+import { writeFile } from 'fs/promises';
+import { Schemas } from '@xata.io/client';
+
+export default class SchemaDump extends BaseCommand {
+  static description = 'Dump the schema as a JSON file';
+
+  static examples = [];
+
+  static flags = {
+    file: Flags.string({ char: 'f', description: 'File to write the schma to' })
+  };
+
+  static args = [];
+
+  async run(): Promise<Schemas.Schema | undefined> {
+    const { flags } = await this.parse(SchemaDump);
+
+    const { databaseURL } = await this.getParsedDatabaseURL();
+    const branchDetails = await getCurrentBranchDetails({ fetchImpl: fetch, databaseURL });
```

Comment:
Another one for https://github.com/xataio/client-ts/pull/291 😄

Reply:
We should change this to:

```suggestion
    const branchDetails = await this.getBranchDetails(databaseURL);
```

Depends which one hets merged before to apply the change.

Reply:
Hahaha. Nice catch. I think I'm going to merge this one first. We can fix this in your PR or another one.